src/Cache.php

```diff
@@ -25,13 +28,15 @@ class Cache
     /**
      * Create new HTTP cache
      *
-     * @param string $type   The cache type: "public" or "private"
-     * @param int    $maxAge The maximum age of client-side cache
+     * @param string      $type   The cache type: "public" or "private"
+     * @param int         $maxAge The maximum age of client-side cache
+     * @param CacheHelper $cache  The cache object to use
```

Comment:
`$cache` can also be `null`:

`@param CacheHelper|null $cache  The cache object to use.`